core/src/main/java/org/jboss/arquillian/persistence/script/splitter/oracle/OracleStatementSplitter.java

```diff
@@ -69,6 +69,11 @@ public void setStatementDelimiter(String statementDelimiter) {
     }
 
     @Override
+    public void setTrimStatementDelimiter(boolean trimStatementDelimiter) {
```

Comment:
This is a bit of a wrong abstraction we have now, but for the moment I don't see how to make it more elegant :)